opendps/command_handler.c

```diff
@@ -130,15 +171,66 @@ static void stat_cmd(uint32_t argc, char *argv[])
     uint32_t v_in = pwrctl_calc_vin(v_in_raw);
     uint32_t v_out = pwrctl_calc_vout(v_out_raw);
     uint32_t i_out = pwrctl_calc_iout(i_out_raw);
-    printf(" V_in  : %2lu.%02lu V\n", v_in/1000,  (v_in%1000)/10);
-    printf(" V_out : %2lu.%02lu V (%s)\n", v_out/1000, (v_out%1000)/10, pwrctl_vout_enabled() ? "enabled" : "disabled");
-    printf(" I_out : %2lu.%03lu A\n", i_out/1000, i_out%1000);
+    power_mode_t mode=ui_get_power_mode();
```

Comment:
If I may be picky, I'd prefer the already set style is followed. I tend to adhere to this at work, even when I don't like it ;) Eg. ```mode = ui_get_power_mode();```